.github/workflows/compute.yml

```diff
@@ -20,6 +20,8 @@ jobs:
   compute:
     name: Rewards
     runs-on: ubuntu-latest
+    permissions: write-all
```

Comment:
Why suddenly?

Reply:
@gentlementlegen I think this is the only thing I would change (remove this line in development) before merging this pull.

Reply:
If this is removed the Action cannot send back the data to the kernel. We need at least `Content: write` permission for it to work. If you don't wanna set it here this has to be set within the repository action settings.

Reply:
I don't understand why this is new. It seems like a major architecture change only in the last month or so? That's why I'm skeptical. What data is being sent back?

Reply:
The data was supposed to be sent back from the start, allowing plugins to be chained and share their outputs. Only a few plugins had the proper `return-data-to-ubiquity-os-kernel` set up, which I fixed when I had to update all of them.